backend/routes/restaurant.js

```diff
@@ -14,7 +14,7 @@ router.get('/', (req, res) => {
     return;
   }
 
-  connection.query(`SELECT * FROM RESTAURANT WHERE ID = ? `, [req.query.restaurantID], (error, results) => {
+  connection.query('SELECT * FROM RESTAURANT WHERE ID = ? ', [req.query.restaurantID], (error, results) => {
```

Comment:
Is there any difference with using `` and ''? I noticed in mysql workbench it generates queries using ``.  @jacoballen4534

Reply:
Messaged jacob personally and it seems its just what the linter wants

Reply:
No, just the linter wants quotes instead of backticks if it is not a template string